radis/test/io/test_geisa.py

```diff
@@ -41,7 +42,24 @@ def test_local_geisa_co(verbose=True, warnings=True, **kwargs):
 
     # 1. Load local file
     fileName = files["CO"]
-    df = gei2df(getTestFile(fileName), cache="regen")
+    df = gei2df(getTestFile(fileName), drop_non_numeric=False, cache="regen")
+    # Post Processing
+    df = add_geisa_local_quanta(df, mol="CO")
+    # Switch 'P', 'Q', 'R' to -1, 0, 1
+    if "branch" in df:
+        replace_PQR_with_m101(df)
+
+    # Ensure branch information exists after post-processing
+    assert "branch" in df.columns
+    assert "jl" in df.columns
+
+    # Validate that all branch values are correctly mapped:
+    # P → -1, Q → 0, R → 1
+    valid_branches = {-1, 0, 1}
+
+    # Check that no unexpected branch values are present
+    assert set(df["branch"].unique()).issubset(valid_branches)
```

Comment:
```suggestion
    assert set(df["branch"].unique()) == set(valid_branches)
```
`issubset` allows missing branches and can hide errors.